Tests/VCRURLConnectionTests/VCRRequestKeyTests.m

```diff
@@ -48,6 +48,73 @@ - (void)testIsEqual {
     XCTAssertEqualObjects(self.key1, self.key2, @"Key objects should be equal");
 }
 
+- (void)testThatRequestsWithDifferentBodiesAreNotEqual {
+    NSString *path = @"http://foo/bar";
+    NSURL *url = [NSURL URLWithString:path];
+    NSMutableURLRequest *request = [[NSMutableURLRequest alloc] initWithURL:url];
+    request.HTTPBody = [@"Omae wa mou shindeiru" dataUsingEncoding:NSUTF8StringEncoding];
+    VCRRequestKey *requestKeyFromURLRequest = [VCRRequestKey keyForObject:request];
+
+    VCRRecording *recording = [[VCRRecording alloc] init];
+    recording.method = @"GET";
+    recording.URI = path;
+    recording.requestData = [@"yamete kudasai" dataUsingEncoding:NSUTF8StringEncoding];
+    VCRRequestKey *requestKeyFromRecording = [VCRRequestKey keyForObject:recording];
+
+    XCTAssertNotEqualObjects(requestKeyFromURLRequest, requestKeyFromRecording, @"Key objects should be equal");
```

Comment:
I think it should "Key objects should not be equal".

Reply:
Good catch, @luisligunas !